internal/monitor/integration_steps_test.go

```diff
@@ -612,6 +629,13 @@ func (i *integration) canLogonToNodesAndDropTestScripts() error {
 		return err
 	}
 
+	if i.isOpenshift {
+		err = i.copyOverTestScriptsToNode(os.Getenv(OpenshiftBastion))
```

Comment:
I think I'm confused. I thought you would already be running the integration test on the bastion, is that true or not?